components/molecules/AlgoliaSearch/components/History.js

```diff
@@ -0,0 +1,123 @@
+import PropTypes from 'prop-types'
+import styles from '../AlgoliaSearch.module.css'
+
+/**
+ * Component for rendering search history.
+ */
+export default function History({
+  history,
+  searchClick,
+  clearLocalStorage,
+  buildSearchUrl
+}) {
+  const config = {
+    now: new Date(), // Now, as a date object
+    nowTs: Date.now(),
+    dayLength: 86400000, // Day in milliseconds.
+    weekLength: 604800000, // Week in milliseconds.
+    months: [
+      'Jan',
+      'Feb',
+      'Mar',
+      'Apr',
+      'May',
+      'Jun',
+      'Jul',
+      'Aug',
+      'Sep',
+      'Oct',
+      'Nov',
+      'Dec'
+    ],
+    days: [
+      'Sunday',
+      'Monday',
+      'Tuesday',
+      'Wednesday',
+      'Thursday',
+      'Friday',
+      'Saturday'
+    ]
+  }
+
+  /**
+   * Compare two date for the same year, month, day.
+   *
+   * @param {*} first
+   * @param {*} second
+   */
+  const isSameDate = (first, second) => {
+    return (
+      first.getFullYear() === second.getFullYear() &&
+      first.getMonth() === second.getMonth() &&
+      first.getDate() === second.getDate()
+    )
+  }
+
+  /**
+   * Convert Timestamp into readable date.
+   *
+   * @param {*} time
+   */
+  const convertDate = (time) => {
```

Comment:
@dcooney Just wanted to point out this project has DayJS loaded. Might be possible to refactor all of your time based functions and leverage https://day.js.org/docs/en/display/format instead?

Reply:
@gregrickaby I'm not seeing Day.js a dep in the project.

Reply:
👀  DayJS is nice! Might be a nice touch to update BRI with this.